### PR TITLE
chore: add deprecated warning to compute_keypair

### PR DESCRIPTION
**Pull request by niuzhenguo**
Adds deprecated warning on docs and link to recomended resource

**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccSomethingV0_basic'
...
=== RUN   TestAccSomethingV0_basic
--- PASS: TestAccSomethingV0_basic (70.75s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       70.796s
```

### CODE REVIEW

File: docs/resources/compute_keypair.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ subcategory: "Elastic Cloud Server (ECS)"
 
 # huaweicloud_compute_keypair
 
+-> **DEPRECATED:**  This resource  has been deprecated. Please use [huaweicloud_kps_keypair](https://www.terraform.io/docs/providers/huaweicloud/r/kps_keypair).
```
Comment (ShiChangkuo) [inline docs/resources/compute_keypair.md:7]: should the `subcategory` be updated to **Deprecated** ?

BTW, this resource is also used in the usage of [cce_node](https://registry.terraform.io/providers/huaweicloud/huaweicloud/latest/docs/resources/cce_node#basic-usage).

Comment (niuzhenguo) [reply docs/resources/compute_keypair.md:7]: just keep here for a few releases before move to Deprecated. will replace the reference of this in the cce node scrpt.

Comment (niuzhenguo) [reply docs/resources/compute_keypair.md:7]: done

**Review by ShiChangkuo — APPROVED** [review]

**Pull request merged**